src/client/cpp/include/nebula/NebulaClient.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef CLIENT_CPP_INCLUDE_NEBULACLIENT_H_
+#define CLIENT_CPP_INCLUDE_NEBULACLIENT_H_
+
+
+#include "ExecuteResponse.h"
+
+namespace nebula {
+namespace graph {
+
+class NebulaClientImpl;
+
+class NebulaClient {
+public:
+    NebulaClient(const std::string& addr, uint16_t port);
+    virtual ~NebulaClient();
+
+    // must be call on the front of the main()
+    static void init(int argc, char *argv[]);
```

Comment:
Please consider that the end user might have multiple instances of NebulaClient. Try not to use global parameters, such as `timeout`.